libs/librtlnumber/verify_librtlnumber.sh

```diff
@@ -46,11 +46,12 @@ for INPUT in ${0%/*}/regression_tests/*.csv; do
 
 		LINE=$((LINE + 1))
 
-		# echo -e "${0##*/}@${HOSTNAME}: DEBUG: LINE: $LINE\n"
-
 		#glob whitespace from line and remove everything after comment
 		input_line=$(echo ${input_line} | tr -d '[:space:]' | cut -d '#' -f1)
 
+		#flip escaped commas to 'ESCAPPED_COMMA' to safeguard agains having them as csv separator
```

Comment:
idk how you missed this typo, you literally typed "escaped" in this comment

Reply:
fixed